frontend/packages/core/src/Feedback/error/dialog.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+import { Dialog, DialogContent } from "../../dialog";
+import type { ClutchError } from "../../Network/errors";
+import Code from "../../text";
+
+interface ErrorDetailsDialogProps {
+  error: ClutchError;
+  open: boolean;
+  onClose: () => void;
+}
+
+const ErrorDetailsDialog = ({ error, open, onClose }: ErrorDetailsDialogProps) => {
+  const prettyPrintError = JSON.stringify(error, undefined, 2);
+
+  return (
+    <Dialog title="Full Error Details" onClose={onClose} open={open}>
```

Comment:
title should be bold

Reply:
this is the Clutch dialog, are you saying it should be bolded everywhere?

Reply:
yeah actually it's weight 500 in the design system too, weight 400 currently in storybook

Reply:
![Screenshot from 2021-03-18 14-56-11](https://user-images.githubusercontent.com/4712430/111689501-1c38f280-87fa-11eb-98e1-a2e6385af0ca.png)